<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineManagementDialog</class>
 <widget class="QDialog" name="EngineManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Chess Engines</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Search:</string>
       </property>
       <property name="buddy">
        <cstring>m_searchEngineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_searchEngineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_clearBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Configured chess &amp;engines:</string>
     </property>
     <property name="buddy">
      <cstring>m_enginesList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="m_enginesList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_addBtn">
         <property name="text">
          <string>&amp;Add...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_configureBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Confi&amp;gure...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_removeBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_enginesList</tabstop>
  <tabstop>m_addBtn</tabstop>
  <tabstop>m_configureBtn</tabstop>
  <tabstop>m_removeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EngineManagementDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EngineManagementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>m_searchEngineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
